src/sources/fluent/mod.rs

```diff
@@ -340,12 +377,37 @@ impl Decoder for FluentEntryStreamDecoder {
     }
 }
 
+struct FluentAcker {
+    chunk: Option<String>,
+}
+
+impl FluentAcker {
+    fn new(frame: &FluentFrame) -> Self {
+        Self {
+            chunk: frame.chunk.clone(),
```

Comment:
If the chunk field is only used for acknowledgements, can we actually assume ownership here instead of cloning it?

Reply:
We could if we owned `frame`, but that is controlled by `trait TcpSource`.